# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libnetfilter_conntrack/0001-include-sys-types.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From cb39932bf20a19b430438f35e3b33d129d3a57af Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Wed, 2 Jan 2013 14:18:24 +0100
Subject: [PATCH] libnetfilter_conntrack.h: include sys/types.h first

We need u_int*_t types.
---
 .../libnetfilter_conntrack.h                       |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/include/libnetfilter_conntrack/libnetfilter_conntrack.h b/include/libnetfilter_conntrack/libnetfilter_conntrack.h
index 12f61d1..0eb8d1c 100644
--- a/include/libnetfilter_conntrack/libnetfilter_conntrack.h
+++ b/include/libnetfilter_conntrack/libnetfilter_conntrack.h
@@ -10,6 +10,7 @@
 #ifndef _LIBNETFILTER_CONNTRACK_H_
 #define _LIBNETFILTER_CONNTRACK_H_
 
+#include <sys/types.h>
 #include <netinet/in.h>
 #include <libnfnetlink/linux_nfnetlink.h>
 #include <libnfnetlink/libnfnetlink.h>
@@ -52,8 +53,6 @@ extern const struct nfnl_handle *nfct_nfnlh(struct nfct_handle *cth);
 
 /* high level API */
 
-#include <sys/types.h>
-
 /* conntrack object */
 struct nf_conntrack;
 
-- 
1.7.2.3

--- ./src/main.c.orig	2013-08-29 02:53:45.692651950 +0200
+++ ./src/main.c	2013-08-29 02:54:35.325781659 +0200
@@ -7,6 +7,7 @@
  * the Free Software Foundation; either version 2 of the License, or
  * (at your option) any later version.
  */
+#include <sys/types.h>
 #include <libnfnetlink/libnfnetlink.h>
 #include <libnetfilter_conntrack/libnetfilter_conntrack.h>
 
